spec/unit/mutant/parser_spec.rb

```diff
@@ -6,16 +6,43 @@
   describe '#call' do
     let(:path) { instance_double(Pathname) }
 
+    let(:expected_node) { s(:sym, :source) }
+
     subject { object.call(path) }
 
+    let(:buffer) do
+      Unparser.buffer(<<~'RUBY')
+        :source # comment_a
+        :source # comment_b
+      RUBY
+    end
+
+    let(:expected_associations) do
+      associations = {}
+      associations.compare_by_identity
```

Comment:
This is forced by parsers comment associator, I'm personally not super in favor to use object memory identity in any kind of API. 

I'd prefer parser would use node index to associate, but I cannot change its API.